packages/web-react/src/components/Pill/stories/Pill.tsx

```diff
@@ -3,9 +3,9 @@ import { ComponentStory } from '@storybook/react';
 import Pill from '../Pill';
 import { SpiritPillProps } from '../../../types';
 
-const Story: ComponentStory<typeof Pill> = <T extends ElementType = 'span'>(args: SpiritPillProps<T>) => (
-  <Pill {...args} />
-);
+const Story: ComponentStory<typeof Pill> = <T extends ElementType = 'span', A = void, E = void>(
```

Comment:
We should better consider the typings API. If the Pill component uses both Action and Emotion colors the developer will not want to extend both of them. He will want to add only custom value/s to the dictionary which is a combination of both.